cli/pkg/providers/minikube/minikube.go

```diff
@@ -45,6 +45,10 @@ func (p *provider) GetOverrides() template.Variables {
 	}
 }
 
+func (p *provider) IsEnvironmentValid() (bool, error) {
+	return true, nil
+}
+
```

Comment:
Since I added this function to the `provider interface` it needs to be implemented for every provider. This makes sense as each install with this new TF approach will require that we first check binaries are on the local machine as well as certain environment variables. I am closing this PR and will open up a new one on the umbrella issue with these changes from your comments. Thanks!